src/Sarif/Core/Tool.cs

```diff
@@ -15,15 +15,15 @@ namespace Microsoft.CodeAnalysis.Sarif
     /// </summary>
     public partial class Tool 
     {
-        public static Tool CreateFromAssemblyData(string prereleaseInfo = null)
+        public static Tool CreateFromAssemblyData(string prereleaseInfo = null, string friendlyName = null)
```

Comment:
Since this is a public API in a public SDK, the design guidelines say to use member overloads instead of default arguments, because default arguments are not CLS-compliant. So:

```C#
public static Tool CreateFromAssemblyData()
{
    return CreateFromAssemblyData(null, null);
}

public static Tool CreateFromAssemblyData(string prereleaseInfo)
{
    return CreateFromAssemblyData(prereleaseInfo, null);
}

public static Tool CreateFromAssemblyData(string prereleaseInfo, string friendlyName)
{
   ...
}
```